app/controllers/concerns/foreman/controller/registration.rb

```diff
@@ -143,4 +141,14 @@ def api_authorization_token
     }]
     User.current.jwt_token!(expiration: 4.hours.to_i, scope: scope)
   end
+
+  def find_organization
+    from_id = Organization.authorized(:view_organizations).find(params['organization_id']) if params['organization_id'].present?
+    from_id || User.current.default_organization || User.current.my_organizations.first
+  end
+
+  def find_location
+    from_id = Location.authorized(:view_locations).find(params['location_id']) if params['location_id'].present?
+    from_id || User.current.default_location || User.current.my_locations.first
```

Comment:
I understand that it's existing logic, so we don't have to change it here, but I think it was a bit cleaner if we required selection of organization, similar to what is done by Katello: https://github.com/Katello/katello/blob/3adebd6507b065dc9c52890ecc93b13beb13c36b/app/controllers/katello/application_controller.rb#L16